cdap-app-fabric/src/main/java/co/cask/cdap/test/RuntimeStats.java

```diff
@@ -34,11 +40,13 @@
 import java.util.Map;
 import java.util.concurrent.TimeUnit;
 import java.util.concurrent.TimeoutException;
+import javax.annotation.Nullable;
 
 /**
  *
  */
-public final class RuntimeStats {
+public final class RuntimeStats implements MetricsConstants {
```

Comment:
Anti-pattern. Shouldn't define constants in interface. `Constants` class should be final class with private constructor.